lib/cog/commands/alias/helpers.ex

```diff
@@ -67,7 +67,7 @@ defmodule Cog.Commands.Alias.Helpers do
   def error({:alias_not_found, alias}),
     do: "I can't find '#{alias}'. Please try again"
   def error(:bad_pattern),
-    do: "Bad pattern. Sorry, but that is an invalid pattern only letters numbers and the following special characters are allowed: *, -, _"
+    do: "Invalid alias name. Only emoji, letters, numbers, and the following special characters are allowed: *, -, _"
```

Comment:
You can actually pass a simple pattern to `ls` using '*' as a wildcard. To me this message implies that you need to pass the entire alias name.